pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerBuilderImpl.java

```diff
@@ -257,6 +258,12 @@ private ProducerBuilderImpl(PulsarClientImpl client, ProducerConfigurationData c
         return this;
     }
 
+    @Override
+    public ProducerBuilder<T> withMultiSchemaMode(ProducerMultiSchemaMode producerMultiSchemaMode) {
+        conf.setMultiSchemaMode(producerMultiSchemaMode);
```

Comment:
Add check for the param producerMultiSchemaMode can't be null.

Reply:
Need not to fix for signature changed